app/assets/stylesheets/home.scss

```diff
@@ -16,4 +16,21 @@
   padding: 2px;
   color: red;
   display: table;
-}
\ No newline at end of file
+}
+
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `random_card_for_check` should be written in lowercase with hyphens